netfields/__init__.py

```diff
@@ -1,5 +1,9 @@
+from django import VERSION
+
 from netfields.managers import NetManager
 from netfields.fields import (InetAddressField, CidrAddressField,
                               MACAddressField)
 
-default_app_config = 'netfields.apps.NetfieldsConfig'
+# only keep it for django 3.1 and below
+if django[0] < 3 or django[0] == 3 and django[1]  < 2:
```

Comment:
`django` is not in scope here:

```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/jfunk/projects/django-postgresql-netfields/.tox/py27-django111/lib/python2.7/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/home/jfunk/projects/django-postgresql-netfields/.tox/py27-django111/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute
    django.setup()
  File "/home/jfunk/projects/django-postgresql-netfields/.tox/py27-django111/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/jfunk/projects/django-postgresql-netfields/.tox/py27-django111/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/home/jfunk/projects/django-postgresql-netfields/.tox/py27-django111/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/lib64/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/jfunk/projects/django-postgresql-netfields/netfields/__init__.py", line 8, in <module>
    if django[0] < 3 or django[0] == 3 and django[1]  < 2:
NameError: name 'django' is not defined
```

Reply:
😊 oh my! sorry, I meant to use `VERSION` of course 😅 fixed in the new version